### PR TITLE
WPB-4559 user receives 2 k federation connection removed on anta

### PR DESCRIPTION
**Pull request by battermann**
https://wearezeta.atlassian.net/browse/WPB-4559

## Checklist

 - [ ] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [ ] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/galley/src/Galley/API/BackgroundProcesses.hs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module Galley.API.BackgroundProcesses where
```
Comment (fisx) [inline services/galley/src/Galley/API/BackgroundProcesses.hs:18]:
```suggestion
module Galley.BackgroundProcesses where
```

not part of the rest api, right?

File: services/galley/src/Galley/API/BackgroundProcesses.hs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import Galley.Effects.TeamStore qualified as E
+import Galley.Types.Conversations.Members (RemoteMember (RemoteMember, rmId))
+import Galley.Types.UserList
+import Imports hiding (head)
+import Polysemy
+import Polysemy.Input
+import Wire.API.Conversation hiding (Member)
+
+-- | Remove all conversations and members from the local domain. This must only be run in a background process
+unsafeRemoveRemoteMembersFromLocalConversation ::
```
Comment (fisx) [inline services/galley/src/Galley/API/BackgroundProcesses.hs:47]: why is it `unsafe`?  because it's not fit for HTTP?  then i'd call it `bg` instead.

**Review by fisx — COMMENTED** [review]
This looks very reasonable, unfortunately it is about to be deprecated by the more robust https://github.com/wireapp/wire-server/pull/3582.

This PR is probably good for code mining when we have a new architecture and implement this again, Galley.API.BackgroundProcesses in particular looks useful.

**Comment by battermann** [comment]
obsolote because of https://github.com/wireapp/wire-server/pull/3582
let's keep the branch though

**Pull request closed without merging**